zephyr/wrapper.c

```diff
@@ -456,26 +456,22 @@ int task_main_start(void)
 
 	/* init self-registered modules */
 	sys_module_init();
+
 	sys_comp_volume_init();
 	sys_comp_host_init();
 	sys_comp_mixer_init();
 	sys_comp_dai_init();
 	sys_comp_src_init();
-	sys_comp_eq_iir_init();
 
-	/* only CAVS18+ have enough memory for these */
-#if defined CONFIG_SOC_SERIES_INTEL_CAVS_V18 ||\
-	defined CONFIG_SOC_SERIES_INTEL_CAVS_V20 ||\
-	defined CONFIG_SOC_SERIES_INTEL_CAVS_V25
-	//sys_comp_mux_init();   // needs more symbols.
 	sys_comp_selector_init();
 	sys_comp_switch_init();
 	sys_comp_tone_init();
 	sys_comp_eq_fir_init();
+	sys_comp_eq_iir_init();
 	sys_comp_keyword_init();
 	sys_comp_asrc_init();
 	sys_comp_dcblock_init();
-#endif
+	sys_comp_mux_init();
```

Comment:
This will require testing on all platforms, particularly APL (which AFAIK is the lowest memory one).

Reply:
So far it loads, I will check should I disable it.

Reply:
If there aren't issues with memory you may be fine.